conda-forge.yml

```diff
@@ -4,3 +4,7 @@ conda_forge_output_validation: true
 github:
   branch_name: main
   tooling_branch_name: main
+provider:
+  linux_aarch64: default
```

Comment:
> Oh I see, travis is pulled for ppc64... I'll remove that.

Nevermind, just saw this. You can add `provider: azure` to have it not run on Travis

Reply:
Done. We'll see how it goes.

Reply:
Note that you don't need to cross-compile (i.e. adding `build_platform: ` _in addition_ to `provider:`). Though the one [CI run](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=610748&view=results) I found with ppc that azure hadn't deleted just was complaining that the compilers weren't being found:
```
configure: C++ compiler search list is: g++ clang++ c++.
configure: *** Could not find a C++ compiler from the search list.
configure: *** Note that a C++ compiler will not be available.
configure: Fortran compiler search list is: gfortran ifort.
configure: *** Could not find a Fortran compiler from the search list.
configure: *** Note that a Fortran compiler will not be available.
```